packages/p2p-v2/src/pages/orders/components/OrderDetailsCard/OrderDetailsCardInfo/PaymentMethodAccordion/PaymentMethodAccordion.scss

```
@@ -0,0 +1,15 @@
+.p2p-v2-payment-method-accordion {
+    &__button {
+        height: 0;
+        padding: 0;
+
+        &:hover {
+            background-color: transparent;
+
+            & > span {
+                // stylelint-disable-next-line declaration-no-important
+                color: #ff444f !important;
+            }
```

@ameerul-deriv, Is this a temporary solution, or is it meant to be long term? 🤔  
```suggestion
            & > span {
                // stylelint-disable-next-line declaration-no-important
                color: #ff444f !important;
            }
```

---

its a temp solution since the buttons that have variant ghost have a hover effect and in this case I don't need the hover effect, i had to add this. But I did raise this to @shayan-deriv already to add another prop to remove any hoverStyles applied to the button 👍 But I'll leave a todo here to remove this once they have added the prop 🙏

---

yes I have a card for it. will address it soon